README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/NativeScript/sample-Groceries.svg?branch=master)](https://travis-ci.org/NativeScript/sample-Groceries)
```

Comment:
When I [view this updated README](https://github.com/ligaz/sample-Groceries/blob/travis-android/README.md) it says “Build unknown”. Will that change once this gets merged into master?